[ADD][16.0] stock_picking_restrict_cancel_printed (#1549)
OCA/stock-logistics-workflow: stock_picking_restrict_cancel_printed/readme/DESCRIPTION.rst

Display printed flag on the form view of the transfer.

> Raise an error when action cancel is called on a printed stock.picking: `You cannot cancel a picking that is already printed.``

Reviewer: ```suggestion
Raise an error when canceling a printed stock transfer: `You cannot cancel a transfer that is already printed.`
```

Author: Thank you! Description updated
